pkg/acsengine/defaults.go

```diff
@@ -98,8 +98,10 @@ func setDefaultCerts(a *api.Properties) (bool, error) {
 
 	ips := []net.IP{firstMasterIP}
 
-	for i := 1; i < a.MasterProfile.Count; i++ {
-		ips = append(ips, net.IP{firstMasterIP[12], firstMasterIP[13], firstMasterIP[14], firstMasterIP[15] + byte(i)})
+	// Include the Internal load balancer as well
+	for i := 1; i < (a.MasterProfile.Count + 1); i++ {
+		ip := net.IP{firstMasterIP[12], firstMasterIP[13], firstMasterIP[14], firstMasterIP[15] + byte(i)}
```

Comment:
Can you remind me what's going on here? `firstMasterIP[12] == first octet` ? When named `firstMasterIP` and why the indexes are `12,13,14,15+i` ?

I realize you're just adapting the existing code, but can you tell me just for my knowledge? Thanks.